package.json

```diff
@@ -30,47 +30,47 @@
   "author": "Webtrekk GmbH",
   "license": "MIT",
   "devDependencies": {
-    "@types/jest": "^24.0.11",
-    "@types/react": "^16.8.8",
-    "@types/react-dom": "^16.8.2",
-    "autoprefixer": "^9.5.0",
+    "@types/jest": "^26.0.10",
+    "@types/react": "^16.9.46",
+    "@types/react-dom": "^16.9.8",
+    "autoprefixer": "^9.8.6",
     "awesome-typescript-loader": "^5.2.1",
     "css-loader": "^2.1.1",
     "dom-testing-library": "^3.17.1",
     "husky": "^2.5.0",
-    "jest": "^24.5.0",
-    "less": "^3.9.0",
+    "jest": "^26.4.0",
+    "less": "^3.12.2",
     "less-loader": "^4.1.0",
-    "lint-staged": "^8.2.1",
+    "lint-staged": "^10.2.11",
     "mini-css-extract-plugin": "^0.5.0",
-    "optimize-css-assets-webpack-plugin": "^5.0.1",
+    "optimize-css-assets-webpack-plugin": "^5.0.3",
     "postcss-loader": "^3.0.0",
     "prettier": "^1.18.2",
-    "react": "^16.8.4",
-    "react-dom": "^16.8.4",
+    "react": "^16.13.1",
+    "react-dom": "^16.13.1",
     "react-testing-library": "^6.0.0",
     "style-loader": "^0.23.1",
-    "ts-jest": "^24.0.0",
+    "ts-jest": "^26.2.0",
     "tslint": "^5.14.0",
     "tslint-config-prettier": "^1.18.0",
     "tslint-react": "^3.6.0",
     "tslint-react-a11y": "^1.0.0",
-    "typescript": "^3.3.3333",
-    "uglifyjs-webpack-plugin": "^2.1.2",
-    "webpack": "^4.29.6",
-    "webpack-cli": "^3.3.0"
+    "typescript": "^3.9.7",
+    "uglifyjs-webpack-plugin": "^2.2.0",
+    "webpack": "^4.44.1",
+    "webpack-cli": "^3.3.12"
   },
   "peerDependencies": {
     "react": "^15.3.0 || 16",
     "react-dom": "^15.3.0 || 16"
   },
   "dependencies": {
-    "@types/lodash": "~4.14.136",
+    "@types/lodash": "^4.14.159",
```

Comment:
They were intentionally set to `~`. What is the reasoning for moving it to `^`?

Reply:
will revert back to tilde